VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
If we need to update this on this release, can we also add it to the `RELEASE_TODO.rst` file?